k2/csrc/fsa.h

```diff
@@ -64,7 +67,7 @@ struct Fsa {
   // Note: an index into the `arcs` array is called an arc-index.
   std::vector<Arc> arcs;
 
-  inline size_t NumStates() { return static_cast<int32_t>(leaving_arcs.size()); }
+  inline std::size_t NumStates() { return static_cast<int32_t>(leaving_arcs.size()); }
```

Comment:
@danpovey, why do we need here to convert `size_t` to `int32_t` and then to `size_t`? instead of just return `leaving_arcs.size()`?

Reply:
no reason, you can remove that.